### ISSUE TITLE
Add new gapless reciter Shaikh Ahmad Naseem

### ISSUE THREAD

**Issue by affohsalif93** [quran/quran_android#1405]
As-salam Alaykum
I found this beautiful recitation at https://www.youtube.com/playlist?list=PLa0tPXx_j_sMTScKRy-HmIzls2fhfpOrU and thought it would be absolutely awesome if you could add it to the app. You may check it and if approved, just let me know so i may cut off the intros. As for the timing files, could you give me an insight on how it's done so that i can contribute to the project.
Jazaka Allahu Khairan

**Comment by nacer80** [comment]
وعليكم السلام ورحمة الله وبركاته 
Mash'Allah he has very good recitation, and good Ahkam tajweed, he is like Sheikh Taniji.

**Comment by affohsalif93** [comment]
Exactly, not too melodious, great pace and like you said his tajweed is wonderful especially for beginners. Let me know how i can help.
Jazaka Allahu Khairan
